src/defaults.rb

```diff
@@ -4,6 +4,11 @@
 
 
 module Metalware
+  # When new defaults are added here you may also want to add these to the
+  # option's help to be displayed.
+  # TODO: Could have these displayed in help automatically by overriding
```

Comment:
What I meant was changing our `Command` class in `CommanderExtensions` so when we call `c.option` in `Cli.rb` the default value, if set, will be automatically included in the help text for the option. But that would be more complicated/magic so I've left it for now.